src/main/java/com/linkedin/replica/serachEngine/models/ResponseType.java

```diff
@@ -0,0 +1,8 @@
+package com.linkedin.replica.serachEngine.models;
+
+/**
+ * Type of responses. 
+ */
+public enum ResponseType {
+	ErrorResponse, SuccessfulResponse
```

Comment:
What about categorizing this according to the status code? For example `BadRequest`, `ServerError`, `Success`, ..etc. ? What do you think?